fs_brokers/apache_hdfs_broker/src/main/java/org/apache/doris/broker/hdfs/FileSystemManager.java

```diff
@@ -91,6 +82,25 @@ public FileSystemManager() {
         writeBufferSize = BrokerConfig.hdfs_write_buffer_size_kb << 10;
         handleManagementPool.schedule(new FileSystemExpirationChecker(), 0, TimeUnit.SECONDS);
     }
+
+    private static String preparePrincipal(String originalPrincipal) throws UnknownHostException {
+        String finalPrincipal = originalPrincipal;
+        String[] components = originalPrincipal.split("[/@]");
+        if (components != null && components.length == 3) {
+            if (components[1].equals("_HOST")) {
+                // Convert hostname(fqdn) to lower case according to SecurityUtil.getServerPrincipal
```

Comment:
delete unuseful comment?

Reply:
I don't  have an environmental to test the case with the principal of the hostname(fqdn), so I think it's better to keep the comment for reminders.